--- wxmacmolplt-7.4.4/resources/wxmacmolplt.desktop~	2010-12-10 00:46:17.000000000 +0100
+++ wxmacmolplt-7.4.4/resources/wxmacmolplt.desktop	2013-05-01 20:39:20.000000000 +0200
@@ -1,10 +1,12 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=wxMacMolPlt
 GenericName=Molecular modelling program
+GenericName[pl]=Program do modelowania molekularnego
 Comment=Plot 3-D molecular structures and normal modes
+Comment[pl]=Wizualizacja modelów molekularnych
 Exec=wxmacmolplt
 Icon=wxmacmolplt
 Terminal=0
 Type=Application
-Categories=3DGraphics;Chemistry;DataVisualization;Graphics;Science;
+Categories=Education;Science;Chemistry;
+# vi: encoding=utf-8
